content/en/blog/2019/webhook/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Secure webhook management
+description: A more secure way to manage Istio webhooks.
+publishdate: 2019-10-29
+attribution: Lei Tang (Google)
+keywords: [security, kubernetes, webhook]
+target_release: 1.4
+---
+
+Istio has two webhooks: Galley and the sidecar injector. Both of them are crucial for Istio,
+with Galley validating Kubernetes resources and the sidecar injector injecting Istio
+sidecar containers.
```

Comment:
Istio has two webhooks: Galley and the sidecar injector. ~Both of them are crucial for Istio,
with~ Galley validates Kubernetes resources and the sidecar injector injects~ing~ sidecar containers into Istio
~sidecar containers~.

Reply:
Revised accordingly.